spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/flyway/FlywayAutoConfiguration.java

```diff
@@ -134,6 +134,7 @@ public Flyway flyway(FlywayProperties properties, DataSourceProperties dataSourc
 			configureFlywayCallbacks(configuration, orderedCallbacks);
 			List<JavaMigration> migrations = javaMigrations.stream().collect(Collectors.toList());
 			configureJavaMigrations(configuration, migrations);
+			configureValidateMigrationNaming(configuration, properties.isValidateMigrationNaming());
```

Comment:
When merging, we should move this down into `configureProperties` and add a comment to note why a method reference isn't being used.

Reply:
Done